libs/ml/tests/ml/layers/fully_connected.cpp

```diff
@@ -28,8 +30,7 @@ class FullyConnectedTest : public ::testing::Test
 {
 };
 
-using MyTypes = ::testing::Types<fetch::math::Tensor<int>, fetch::math::Tensor<float>,
-                                 fetch::math::Tensor<double>,
+using MyTypes = ::testing::Types<fetch::math::Tensor<float>, fetch::math::Tensor<double>,
```

Comment:
actually I think we can handle Tensor<int> here as well.
The only change that needs to be made is to add to math/base_types a method for function tolerance:
fetch::math::meta::IfIsInteger<T, T> function_tolerance()
{
  return T(0);
}